tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -0,0 +1,110 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+    <TargetExt>.dll</TargetExt>
+    <RuntimeIdentifiers>win-x86;win-x64;linux-x64;osx-x64</RuntimeIdentifiers>
+    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp3.1</TargetFrameworks>
```

Comment:
Nit: can you move this line up next to the original `<TargetFrameworks>` line?  It's easier to find.